sys/kern/tty.c

```diff
@@ -944,9 +929,7 @@ static int dev_tty_open(vnode_t *v, int mode, file_t *fp) {
   return error;
 }
 
-vnodeops_t dev_tty_vnodeops = {.v_open = dev_tty_open,
-                               .v_getattr = tty_vn_getattr,
-                               .v_access = vnode_access_generic};
+vnodeops_t dev_tty_vnodeops = {.v_open = dev_tty_open};
```

Comment:
Should that not be `static`?

Reply:
Sure, why not. It's not exposed in the header anyway